app/helpers/email_signup_helper.rb

```diff
@@ -5,9 +5,10 @@ def email_signup_path(atom_feed_url)
     if mhra_exists_for_atom_feed?(atom_feed_url)
       mhra_email_signup_path_from_atom_feed(atom_feed_url)
     elsif feed_uri.path.match(%r{^/world/(.*)\.atom$})
-      new_email_signups_path(email_signup: { feed: atom_feed_url })
```

Comment:
In the PR description you say that this shouldn't be merged until non english World Location News pages are rendered by Collections. The Commit message says this needs to be done because we're still rendering non English WL News pages in Whitehall. 

Given that this is dependent on our next migration, can this method just be removed if it's only for non english world location news pages

Reply:
I've now realised the entire `EmailSignupHelper` can be removed (which I've done in [this commit(https://github.com/alphagov/whitehall/pull/6950/commits/8dbea2de034ce4e99db83f7746deafdb1f2319cc)).

That then led me to discover some partials that weren't being used, which have been removed in [this commit](https://github.com/alphagov/whitehall/pull/6950/commits/1d785d8870ccf5ace4e47cf017e22ec0f5ab4930).

Another 144 lines of code deleted 🎉!